src/main/java/bio/terra/workspace/app/configuration/external/TracingConfiguration.java

```diff
@@ -21,11 +23,14 @@ public void setProbability(Double probability) {
     this.probability = probability;
   }
 
-  public String getProjectId() {
```

Comment:
This is a small drive-by change: I don't think we need to be manually specifying a tracing project ID here. We should use the default project ID (from app default credentials) for tracing, and Google provides reasonable helper methods to extract this.

Reply:
Is this always supposed to be the project where the service is running?

Reply:
That is, TDR and WSM work across many projects. We don't expect this ID to change for that. If we want to keep track of the project we are working on, we log that separately. Yes?

Reply:
Yes – this line was only configuring the target Google Project that OpenCensus would export service-level trace information to, which is (as far as I can tell) 100% of the time the Google Project that's hosting the k8s cluster.